src/resonanceReconstruction.hpp

```diff
@@ -52,7 +52,7 @@ namespace ENDF = ENDFtk::resonanceParameters;
 #include "resonanceReconstruction/ZeroWidth.hpp"
 
 #include "resonanceReconstruction/breitWigner.hpp"
-#include "resonanceReconstruction/reichMoore.hpp"
+//#include "resonanceReconstruction/reichMoore.hpp"
```

Comment:
I can take out the comment for now till you're ready in reconr. I commented it since I do not need it anymore and to test the compilation time with gcc.

Reply:
When we decide this is the way to go, we should remove all the other reichMoore files/namespace.

Reply:
Of course, but I did not want to do that just yet.